geopandas/geoseries.py

```diff
@@ -1051,7 +1075,7 @@ def estimate_utm_crs(self, datum_name="WGS 84"):
         """
         return self.values.estimate_utm_crs(datum_name)
 
-    def to_json(self, **kwargs):
+    def to_json(self, **kwargs) -> dict:
```

Comment:
```suggestion
    def to_json(self, **kwargs) -> str:
```
This returns JSON string.